src/main/java/bio/terra/workspace/common/utils/MdcHook.java

```diff
@@ -128,4 +128,11 @@ private String serializeMdc(@Nullable Map<String, String> mdcMap) {
     }
     return mdcContext;
   }
+
+  @Override
+  public HookAction stateTransition(FlightContext context) throws InterruptedException {
+    logger.info(
```

Comment:
The log level might benefit from being set per-hook or even per-flight class instead of per-statement. I could see this becoming noisy in high traffic flights.